### DISCUSSION TITLE
Resume Paddle subscription after deactivating Proton Unlimited

### DISCUSSION THREAD

**Discussion by iMestie** [simple-login/app#1574] [Q&A]
Hi, sorry, I wasn't able to find an answer to this question. I hope I didn't miss it among the discussions.

I'm an existing SimpleLogin Premium user (billed via Paddle) and I was thinking of subscribing to Proton Unlimited. That way, by linking the two services, I would have SL included in the offer.
My main concern is: what happens if I decide to downgrade or cancel my Proton subscription? Am I able to resume Paddle payments and keep all my aliases working? I'm afraid SL would bring me down to the Free tier and that all my aliases stop working or, even worse, get canceled.
Could someone please explain how this integration work?

Thank you in advance! :)

**Comment by nguyenkims** [comment]
@iMestie hi you can more info about how proton subscription is used in SL on https://simplelogin.io/docs/subscription/switch-plan-proton-unlimited/
The TLDR is unfortunately the Paddle and Proton subscriptions aren't cumulable and when you upgrade to Proton Unlimited, the Paddle one isn't paused ...

**Reply by iMestie** [reply]
> @iMestie hi you can more info about how proton subscription is used in SL on https://simplelogin.io/docs/subscription/switch-plan-proton-unlimited/ The TLDR is unfortunately the Paddle and Proton subscriptions aren't cumulable and when you upgrade to Proton Unlimited, the Paddle one isn't paused ...

Oh no thanks @nguyenkims, I got that! My question was different, I meant the other way around. Can I get back to a Paddle subscription, if I decide to terminate my Proton Unlimited plan? The link you provided offers no explanation about that. :)

**Reply by nguyenkims** [reply]
ah I see. Yes of course, you can always come back to Paddle as soon as your Proton subscription finishes.

**Reply by iMestie** [reply]
> ah I see. Yes of course, you can always come back to Paddle as soon as your Proton subscription finishes.

Perfect, thanks!
